lib/scidata/caltech101.ex

```diff
@@ -138,8 +138,10 @@ defmodule Scidata.Caltech101 do
   end
 
   defp download_dataset(_dataset_type, opts) do
+    base_url = opts[:base_url] || @base_url
```

Comment:
Let's also break this one into `base_url` and `dataset_file`.

Reply:
No problem :) And I've also updated the inline docs.